circleci/utils

```diff
@@ -21,6 +21,29 @@ install_awscli(){
   echo "Completed AWS cli install"
 }
 
+assume_role_with_web_identity() {
+  echo "Setting aws profile to $2"
+  aws configure set profile $2
+  echo "Authenticating via role $1..."
+  aws sts assume-role-with-web-identity \
+    --role-arn "$1" \
+    --role-session-name "oidc-ecr-role-session" \
+    --web-identity-token "${CIRCLE_OIDC_TOKEN_V2}" \
+    --query 'Credentials.[AccessKeyId,SecretAccessKey,SessionToken]' \
+    --output text > /tmp/cred.txt
+  read -r AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN < /tmp/cred.txt 
```

Comment:
Writing this out to a temporary file and then reading to environment variable seems risky, given that it's not cleaned up.

Can we just write them directly to the environment variables?